.github/CODEOWNERS

```diff
@@ -1457,6 +1457,7 @@ packages/kbn-monaco/src/esql @elastic/kibana-esql
 /x-pack/test/api_integration/services/ml.ts @elastic/ml-ui
 
 # Additional plugins and packages maintained by the ML team.
+/x-pack/test/functional/es_archives/large_arrays @elastic/ml-ui # Assigned per usages
```

Comment:
Nit - for future maintainability as these sections grow in size, could you move this under `/x-pack/test/functional/apps/transform/`

Reply:
https://github.com/elastic/kibana/pull/200946/commits/5bad6de322b712cefa2937c034d3529b21c79a52